server.js

```diff
@@ -2,9 +2,20 @@ const net = require('net');
 const JsonSocket = require('json-socket');
 
 
-// Object containing clients list and relative information as such
-// {clientId: {socket: ..., charactersMaxLength: ...}, ...}
-const clients = {};
+const MAX_CONCURRENT_JOBS = 50;
```

Comment:
@VincenzoFerme this will need persiseance in MongoDB or something. The guys can use something like https://github.com/rschmukler/agenda if they want to.